### DISCUSSION TITLE
Installed page rework

### DISCUSSION THREAD

**Discussion by cassidyjames** [elementary/appcenter#1658] [Ideas]
- Split updates from installed layout-wise (grid for installed, list for updates)?
- Maybe pack release changes below icon and button (related: #1654)
- Maybe clamp the list?

**Comment by elsiehupp** [comment]
Might a FlowBox work better than a grid for the purpose of adaptive window resizing? If the installed/updates list items are a reusable widget class also appearing with subtle variations other app-list areas of the app, consistently using FlowBoxes could make the different sections feel extra cohesive.

And maybe something like a [HdyLeaflet](https://blogs.gnome.org/alexm/2019/12/12/gnome-and-gestures-part-3-hdyleaflet-again/) providing a list of sections could be nice? Though it would be a fairly substantial rework. (This would probably be more useful on the home page, though, for when you click into a category, since that’s more of a “drill-down” interface than the “installed” list.)

**Reply by cassidyjames** [reply]
Yes, I meant grid as in a grid layout, but not necessarily a `Gtk.Grid` widget. 😅 Flowbox like we're using now in other contexts could make sense.

**Comment by danirabbit** [comment]
Gonna close this since it seems like everything here was resolved
